models/zendesk__ticket_enriched.sql

```diff
@@ -82,15 +85,20 @@ with ticket as (
         requester_updates.last_updated as requester_ticket_last_update_at,
         requester.last_login_at as requester_last_login_at,
         requester.organization_id as requester_organization_id,
+        {% if var('using_organizations', True) %}
         requester_org.name as requester_organization_name,
+        {% endif %}
 
         --If you use organization tags this will be included, if not it will be ignored.
-        {% if var('using_organization_tags', True) %}
+        {% if var('using_organization_tags', True) and var('using_organizations', True) %}
         requester_org.organization_tags as requester_organization_tags,
         {% endif %}
+        --If you use organizations this will be included, if not it will be ignored.
+        {% if var('using_organizations', True) %}
```

Comment:
Perhaps we can combine with line 89 into just 1 config?

`requester_org.name as requester_organization_name,`

Reply:
Hmm good question. Does changing the order of fields change the schema? I wouldn't want to do that if so since that's a breaking change. Otherwise, happy to consolidate.

Reply:
Good point! According to G, it shouldn't be a breaking change as long as the names and datatypes don't change since that's how a schema gets defined by. But I will let you decide since it's a really minor improvement

Reply:
I think I'll keep it as is, even if it'd simplify the model code--just from a table design perspective, it will be easier for customers to see `requester_organization_name` first rather than `requester_organization_tags` to  best understand requester organization information.

Reply:
Makes sense!